src/Tizen.Multimedia/MediaTool/MediaPacket.cs

```diff
@@ -656,6 +657,22 @@ public static MediaPacket Create(MediaFormat format)
             return new SimpleMediaPacket(format);
         }
 
+        /// <summary>
+        /// Creates an object of the MediaPacket based on the original MediaPacket and increase ref count of both MediaPacket instances.
+        /// </summary>
+        /// <remarks>
+        /// After this instance creation, the internal data of both MediaPacket instances are the same and those ref count is greater than the original MediaPacket by 1.\n
```

Comment:
"those" need to be changed, if it references "MediaPacket instances" then "theirs" would be better, if it references "internal data" then some restructuring would be in place.

Reply:
How about this?
```
This method can be useful when user share MediaPacket instance to other module and don't know the exact time to dispose it.\n
In this case, user create a new instance with <see cref="Create(MediaPacket)"/> and share it to other module.\n
And then, each MediaPacket instances can be disposed independently without concerning other instance's life cycle.
```

Reply:
great, just change "user create ... and share" -> "user creates ... and shares"